src/runtime/manifest-parser.pegjs

```diff
@@ -1833,9 +1854,14 @@ PrimaryExpression
   {
     return toAstNode<AstNode.BuiltInNode>({kind: 'built-in-node', value: fn});
   }
-  / fn: fieldName
+  / fn: fieldName nested:('.' fieldName)*
   {
-    return toAstNode<AstNode.FieldNode>({kind: 'field-name-node', value: fn});
+    // nested is ignored, used only to allow Paxel expressions to parse as text
+    if (!isPaxelMode() && nested && nested.length > 0) {
+      error('Scope lookups are not permitted for refinements, only in paxel expressions.');
+    } else {
+      return toAstNode<AstNode.FieldNode>({kind: 'field-name-node', value: fn});
```

Comment:
where is `nested` appearing here?

Also, I think the AST in Kotlin for a.b.c is:
```
FieldExpr(
  var: "c",
  qualifier =  FieldExpr(
    var: "b",
    FieldExpr(
      var: "a",
      qualifier: null
    )
  )
)
```

You probably **don't want** it the other way around here, i.e.
```
FieldExpr(
  var: "a",
  qualifier =  FieldExpr(
    var: "b",
    FieldExpr(
      var: "c",
      qualifier: null
    )
  )
)
```

Reply:
Oh, sorry, I've read the comment... we don't care about AST structure here yet. Ignore me.